javascript-if/main.js

```diff
@@ -0,0 +1,69 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  }
+  return false;
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  }
+  return false;
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'j') {
+    return true;
+  }
+  return false;
+}
```

Comment:
This should only return `true` if `string` begins with a _capital_ "J", not a _lowercase_ "j".

---

![image](https://user-images.githubusercontent.com/16143771/169148881-7c6d5a77-23d6-40e2-ab04-925b585284a7.png)